distribution/src/config/discovery-file/unicast_hosts.txt

```diff
@@ -3,18 +3,23 @@
 # mechanism.  This file should contain one entry per line, where an entry is a
 # host/port combination.  The host and port should be separated by a `:`.  If
 # the port is left off, a default port of 9300 is assumed.  For example, if the
-# cluster has three nodes that participate in the discovery process:
-# (1) 66.77.88.99 running on port 9300 (2) 66.77.88.100 running on port 9305
-# and (3) 66.77.88.99 running on port 10005, then this file should contain the
+# cluster has three nodes that participate in the discovery process: (1)
+# 66.77.88.99 running on port 9300 (2) 66.77.88.100 running on port 9305 and
+# (3) 66.77.88.99 running on port 10005, then this file should contain the
 # following text:
 #
 #10.10.10.5
 #10.10.10.6:9305
 #10.10.10.5:10005
 #
-# For IPv6 addresses, make sure to put a bracket around the host part of the address,
-# for example: [2001:cdba:0000:0000:0000:0000:3257:9652]:9301 (where 9301 is the port).
+# For IPv6 addresses, make sure to put a bracket around the host part of the
+# address, for example: [2001:cdba:0000:0000:0000:0000:3257:9652]:9301 (where
+# 9301 is the port).
+#
+# NOTE: all lines starting with a `#` are comments, and comments must exist on
+# lines of their own (i.e. comments cannot begin in the middle of a line)
+#
+# NOTE: Set `discovery.zen.hosts_provider: file` to enable file-based discovery
```

Comment:
NB this file is now included in all distributions, rather than just in the plugin. The only change, apart from reformatting, is this `NOTE`.